<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectWindow</class>
 <widget class="QMainWindow" name="ProjectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>357</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Project Window</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Project information</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="margin">
         <number>2</number>
        </property>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="horizontalSpacing">
           <number>2</number>
          </property>
          <property name="verticalSpacing">
           <number>2</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="txtNombreProyecto"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Owner</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="txtPropietarios"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Address</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Notes</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="txtDireccion"/>
          </item>
          <item row="5" column="1">
           <widget class="QTextEdit" name="txtNotas"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="Telefono">
            <property name="text">
             <string>Telephone</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="txtTelefono"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>e-mail</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="txtEMail"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabProyecto">
       <attribute name="title">
        <string>Building parts</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="margin">
         <number>2</number>
        </property>
        <item>
         <widget class="QTreeWidget" name="treeProyecto">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Material</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Shape</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Dimensions</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Quantity</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Measure Unit</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabComputo">
       <attribute name="title">
        <string>Material Ammounts</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="margin">
         <number>2</number>
        </property>
        <item>
         <widget class="QTreeWidget" name="treeComputoMetrico">
          <column>
           <property name="text">
            <string>Part-Material</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Quantity</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Unit</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="treeComputoMateriaPrima">
          <column>
           <property name="text">
            <string>Material</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Quantity</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Unit</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRemove_Build_Part">
   <property name="text">
    <string>Remove Build Part</string>
   </property>
   <property name="toolTip">
    <string>Removes selected Build Part</string>
   </property>
  </action>
  <action name="actionClone_and_set_material">
   <property name="text">
    <string>Clone and set material</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
